app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Parent exception for all vaccine-related issues."""
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """Raised when a visitor is not vaccinated."""
+    pass
```

Comment:
As per the checklist, you should provide descriptive error messages for your custom exceptions. Right now, your exceptions do not provide any error messages.